pkg/profiler/cpu/maps.go

```diff
@@ -138,3 +144,92 @@ func (m *bpfMaps) clean() error {
 
 	return nil
 }
+
+// setUnwindTable updates the unwind tables with the given unwind table.
+func (m *bpfMaps) setUnwindTable(pid int, ut unwind.UnwindTable) error {
+	buf := new(bytes.Buffer)
+
+	// Write number of rows `.table_len``.
+	if err := binary.Write(buf, m.byteOrder, uint64(len(ut))); err != nil {
+		return fmt.Errorf("write the number of rows: %w", err)
+	}
+
+	if len(ut) >= maxUnwindTableSize {
+		panic(fmt.Sprintf("max unwind table size reached. Table size %d, but max size is %d", len(ut), maxUnwindTableSize))
```

Comment:
:+1:

Reply:
You know my view on the panics. This will halt the whole agent. I would rather fail ~~this specific profiler~~(we don't have multiple profilers anymore) or prevent profiling this particular process and log visible error messages.

Reply:
Here I agree, a panic is not a good fit, will remove it from here!